### PR TITLE
Readme clarifications

### PR DESCRIPTION
**Pull request by leeN**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -26,41 +28,44 @@ The "Foxhound" browser can be built mostly by following instructions on how to b
...
 ${HOME}/.cargo/bin/rustup install 1.66
 ${HOME}/.cargo/bin/rustup default 1.66
 ${HOME}/.cargo/bin/rustup override set 1.66
 ```
 
-To install version ```1.66``` of the rust compiler. You need to start a new shell for the changes to take effect.
-
-## Compiling
+To install version `1.66` of the rust compiler. You need to start a new shell for the changes to take effect.
+The [Firefox documentation](https://firefox-source-docs.mozilla.org/writing-rust-code/update-policy.html#schedule) provides information about the matching rust version for each Firefox release.
```
Comment (tmbrbr) [inline README.md:53]: Thanks for adding this link!

**Review by tmbrbr — APPROVED** [review]
Looks great, thanks for this!

**Review by tmbrbr — APPROVED** [review]

File: README.md [hunk 2]
```diff
@@ -114,7 +128,7 @@ cd js/src
 ./tests/jstests.py ../../obj-tf-release/dist/bin/js taint/
 ```
 
-## Citations
+## Cite us!
 
 If you would like to use project Foxhound in your own research, please cite our EuroS&P paper where we describe the browser:
```
Comment (tmbrbr) [inline README.md:133]: I suggest we also add a comment to open an issue once the paper has been accepted so we can update the wiki.

Comment (leeN) [reply README.md:133]: Good idea, will do tomorrow

Comment (leeN) [reply README.md:133]: Completely forgot about this. Last push adds this :)

**Pull request merged**